internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1214,6 +1214,13 @@ func (r *ClusterDeploymentsReconciler) TransformClusterToDay2(
 	if err != nil {
 		log.WithError(err).Errorf("failed to transform cluster %s into day2 cluster", cluster.ID.String())
 	}
+	// Set custom Ignition endpoint so that day2 workers would join using HTTPS endpoint
+	// ensureMCSCert would add ignition override with MCS secret
+	ignitionEndpoint := fmt.Sprintf("https://api-int.%s.%s:22623/config/%s", cluster.Name, cluster.BaseDNSDomain, models.HostRoleWorker)
```

Comment:
maybe better to use 
cluster.APIVipDNSName as api-int.%s.%s ?

Reply:
but maybe it is not really critical

Reply:
I agree, it should be api-int. I thought it was api-int. It's not:

ZTP transform

https://github.com/openshift/assisted-service/blob/c45f3699d83665a5e27a00939a9ff40907f39111/internal/cluster/cluster.go#L1347

UI OCM import

https://github.com/openshift-assisted/assisted-ui-lib/blob/61aca82c9edc9f2b9f914a1baec92dffc9eb5926/src/ocm/services/Day2ClusterService.ts#L31

Reply:
UI OCM import

https://github.com/openshift-assisted/assisted-ui-lib/blob/61aca82c9edc9f2b9f914a1baec92dffc9eb5926/src/ocm/services/Day2ClusterService.ts#L31